pages/docs/reference/functions.md

```diff
@@ -177,10 +178,11 @@ fun foo(vararg strings: String) { /*...*/ }
 
 foo(strings = *arrayOf("a", "b", "c"))
 ```
+
 </div>
 
-> **On the JVM**: the named argument syntax cannot be used when calling Java functions because Java bytecode does not
-always preserve names of function parameters.
+> **On the JVM**: You can't use the named argument syntax when calling Java functions because Java bytecode does not
+always preserve names of function arguments.
```

Comment:
Parameter is the correct term here. Parameter refers to the definition of the variable in the function header while an argument is the value passed to a a function. 
```suggestion
always preserve names of function parameters.
```

Reply:
Let's keep parameters. Thanks!